packages/react-components/src/components/timegraph-output-component.tsx

```diff
@@ -288,6 +298,59 @@ export class TimegraphOutputComponent extends AbstractTreeOutputComponent<Timegr
         }
     }
 
+    private findElement(payload: { [key: string]: string | { [key: string]: string }}): TimeGraphEntry | number | undefined {
+        const startTimestampString: string = payload['startTimestamp'].toString();
+        const found = this.state.timegraphTree.findIndex(e => e.start === BigInt(startTimestampString));
```

Comment:
The entry start time in not relevant in most cases not matching the selected event time. So, you can remove it.

Reply:
Done.

Reply:
Done.